server/api/utils/oauth_helper.py

```diff
@@ -0,0 +1,85 @@
+import base64
+from google_auth_oauthlib.flow import Flow
+from django.conf import settings
+from Crypto.Cipher import AES
+
+from Crypto import Random
+IV = Random.new().read(AES.block_size)
```

Comment:
### Nitpick

I'm not sure if we are using `flake8` or `pep8` but I believe you should leave two line breaks after import statements, for readability purpose